templates/resources.jsx

```diff
@@ -42,6 +44,36 @@ export default function Resources (props) {
     return (_forceDownload || filename);
   }
 
+  function onFilterClicked(e) {
+    if (e && e.preventDefault) e.preventDefault();
+
+    const $clickedButton = this.$(e.currentTarget);
+    const type = $clickedButton.data('filter');
+
+    updateSelectedType(type);
+
+    // const clickedTabId = $clickedButton.attr('id');
+
+    // this.$('.js-resources-filter-btn-click').removeClass('is-selected').attr('aria-selected', false);
+
+    // $resources.attr('aria-labelledby', clickedTabId);
+    // $clickedButton.attr('aria-selected', true);
+
+    // let items;
+    // const filter = $clickedButton.addClass('is-selected').attr('data-filter');
+    // if (filter === 'all') {
+    //   items = this.$('.js-resources-item').removeClass('u-display-none');
+    // } else {
+    //   this.$('.js-resources-item')
+    //     .removeClass('u-display-none').not('.is-' + filter)
+    //     .addClass('u-display-none');
+    //   items = this.$('.js-resources-item.is-' + filter);
+    // }
+
+    // if (items.length < 0) return;
+    // a11y.focusFirst($(items[0]));
```

Comment:
does `a11y.focusFirst` need to be called?

Reply:
Thanks, Chris.  Sorry, my last push was a work in progress, and then I was out on PTO on Friday.  I'll pick this back up shortly.